Resources/Prototypes/_LostParadise/Entities/Markers/Spawners/Random/Salvage/spawners.yml

```
@@ -0,0 +1,495 @@
+    - id: MobCarpSalvage
+      weight: 15
+      amount: !type:RangeNumberSelector
+        range: 1, 3
+    - !type:AllSelector
+      weight: 5
+      children:
+      - id: MobSharkSalvage
+      - id: MobCarpSalvage
+        amount: !type:ConstantNumberSelector
+          value: 2
+    # - id: MobHivebot (solo hivebot spawn)
```

yamllint (1.35.1)

[warning] comment not indented like content

(comments-indentation)